packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/list/OpenstackCredentialsList.tsx

```diff
@@ -97,7 +98,8 @@ export const OpenstackCredentialsList: React.FC<ListComponentProps> = ({ secret,
         helperTextPopover: (
           <Trans t={t} ns="plugin__forklift-console-plugin">
             Note: If this field is checked/true, the migration from this provider will be insecure.
-            {'<br><br>'} Insecure migration means that the transferred data is sent over an insecure
+            <br />
+            <br /> Insecure migration means that the transferred data is sent over an insecure
```

Comment:
<br> on the first line and then on the empty line

```
line<br>
<br>
next line
```

Reply:
lint:fix switches back to
< br/ >
< br/ >